assets/css/gdpr-public-custom.css

```diff
@@ -0,0 +1,8 @@
+.gdpr-consent-management.gdpr-active .gdpr-info .gdpr-cookies-used .gdpr-cookies span a {
```

Comment:
Do these lines work the same without that `span`?

Reply:
I don't see why they wouldn't, will update

Reply:
Updated